CHANGELOG.md

```diff
@@ -67,6 +72,9 @@ Other minor additions
   AlmostCancellative      e _•_ = AlmostLeftCancellative e _•_ × AlmostRightCancellative e _•_
   ```
 
+* Added export of `+-*-commitativeSemiring` to the record of
+  of `Algebra.Bundles.CancellativeCommutativeSemiring`.
```

Comment:
This entry is not needed as you added `CancellativeCommutativeSemiring` in this version of the library.

Reply:
Removed.
This is needed to remove - if PR is considered relatively to the last official lib version (say, 1.4).
But if it is considered  relatively to the current `master`, then its is needed to stay - because it changes the record of
 CancellativeCommutativeSemiring.
(?)

Reply:
If you take a moment to think about it, I'm pretty sure you'll agree with me that it wouldn't make much sense to have a CHANGELOG entry for every change relative to the current state of master!

Reply:
Probably  CHANGELOG.md in each my commit  needs to list the changes relatively to the last official library release,
and for this GCD PR, this release is lib-1.4.
Right?